presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -1061,7 +1061,13 @@ standard rules for nulls. The subquery must produce exactly one column::
 
     SELECT name
     FROM nation
-    WHERE regionkey IN (SELECT regionkey FROM region)
+    WHERE regionkey IN (SELECT regionkey FROM region WHERE name = 'AMERICA' OR name = 'AFRICA')
```

Comment:
This is too wide for the default HTML rendering (makes the box scroll). Either wrap `WHERE` to the next line (aligned with the subquery `SELECT`) or shorten the expression